.github/workflows/ci.yml

```diff
@@ -6,6 +6,62 @@ on:
   pull_request:
 
 jobs:
+  linux-build:
+    name: Build and Test
+
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Cancel previous
+      uses: styfle/cancel-workflow-action@0.9.1
+      with:
+        access_token: ${{ github.token }}
+
+    - name: Install build dependencies for simview and surelog
+      run: |
+        sudo apt-get update -qq
+        sudo apt install -y \
+          build-essential cmake ninja-build \
+          g++ default-jre \
+          google-perftools \
+          libgoogle-perftools-dev \
+          uuid-dev
+
+    - name: Setup Python for Surelog build
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.8
+        architecture: x64
+
+    - name: Setup Python Packages
+      run: |
+        pip3 install orderedmultidict
+        pip3 install psutil
+
+    - name: Configure shell
+      run: |
+        echo 'PREFIX=${GITHUB_WORKSPACE}/install' >> $GITHUB_ENV
+
+    - name: Build Surelog
+      run: |
+        cd ${GITHUB_WORKSPACE}
+        git clone --recursive https://github.com/chipsalliance/Surelog.git
+        (cd Surelog ; make install)
+
+    - name: Checkout simview
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+
+    - name: Build
+      run: |
+        cmake -B build -GNinja -DCMAKE_PREFIX_PATH=$PREFIX .
```

Comment:
Is prefix needed for the CI?

Reply:
yes, because we are installing Surelog not in the /usr/local/... but into our own chosen prefix directory. Now we have to tell cmake where that is, so that it can find the relevant surelog dependency.

Reply:
Makes sense. I'd like to set up simview with surelog as a git sub module, instead of requiring it to be locally installed. A change in surelog api can cause breakages at the moment.

Reply:
Yes, I think that would make for a more robust experience.